src/services/doppler-billing-api-client.ts

```diff
@@ -71,6 +72,7 @@ export class HttpDopplerBillingApiClient implements DopplerBillingApiClient {
       currency: x.currency,
       amount: x.amount,
       filename: x.filename,
+      downloadInvoiceUrl: x.downloadInvoiceUrl,
```

Comment:
I have no preference for your internal name, but in the billing API I would prefer something more concise. What do you think about `downloadUrl`, `fileUrl` or `file`?

Another option is going back to our initial approach and use a dynamic `_links` field with a descriptive `rel` property: https://github.com/FromDoppler/doppler-billing-api/commit/4411b0a62f460b0c2cbfedc9afd446f283ec560f#diff-04c6e90faac2675aa89e2176d2eec7d8L26-R26

Reply:
I like the idea to use a dynamic links property instead of a new property for the download invoice url. At the moment the property can have:

links: [{
              "rel": "downloadUrl",
               "path": "/accounts/doppler/22235/invoices?from=2020-02-01T00:00:00Z&to=2020-07-01T00:00:00Z"
           }]

Do you agree?

Reply:
Yes, but I think that we should use absolute URLs, so:

```json
{
  // . . .
  "_links": [
    {
      "rel": "downloadUrl",
      "path": "https://apis.fromdoppler.com/billing-api/accounts/doppler/22235/invoices/invoice_2020-10-05_10.pdf?0bd058e8fa5788b889dedf74ab900d5e51532d81"
    }
  ]
}
```

Reply:
Notice that I have replaced `links` by `_links` (to denote metadata), `path` by `href`, and the `rel` value by `file`; also added a `description` field.

Reply:
yes, perfect. We will implement it.

Reply:
Done. I updated the code to implment it also I added the unit tests for the different scenarios.